src/app/pages/RewardPage/components/ClaimForms/RewardClaimForm/hooks/useClaimRewardSov.tsx

```diff
@@ -4,53 +4,29 @@ import { useAccount } from 'app/hooks/useAccount';
 import { ethGenesisAddress, gasLimit } from 'utils/classifiers';
 import { TxType } from 'store/global/transactions-store/types';
 
-export const useClaimRewardSov = (
-  hasLockedSov: boolean,
-  hasLMRewards: boolean,
-) => {
+export const useClaimRewardSov = () => {
   const address = useAccount();
 
-  const { send: sendLocked } = useSendContractTx(
-    'lockedSov',
-    'createVestingAndStake',
-  );
-
   const { send, ...tx } = useSendContractTx(
     'liquidityMiningProxy',
     'claimRewardFromAllPools',
   );
 
   return {
     send: async () => {
-      let nonce = await contractReader.nonce(address);
-      if (hasLockedSov) {
-        await sendLocked(
-          [],
-          {
-            nonce,
-            from: address,
-            gas: gasLimit[TxType.LOCKED_SOV_CLAIM],
-          },
-          {
-            type: TxType.LOCKED_SOV_CLAIM,
-          },
-        );
-        nonce++;
-      }
+      const nonce = await contractReader.nonce(address);
```

Comment:
You can remove this. Retrieving nonce is required only when making multiple transactions (for example when trying to have token approval & deposit tx'es to be confirmed in the same block)